wg-data-protection/charter.md

```diff
@@ -0,0 +1,76 @@
+# WG Data Protection Charter
+
+This charter adheres to the [wg-governance] guidance as well as
+the general conventions described in the [Kubernetes Charter README] and
+the Roles and Organization Management outlined in [sig-governance], where
+applicable to a Working Group.
+
+
+## Scope
+
+Data protection in Kubernetes context typically involves backup and recovery
+of two types of entities:
+* Kubernetes object resources
+* Persistent volume data
+We consider it a complicated and layered problem, including backup and recovery
+at persistent volume level, application level, and cluster level. Part of the
+working group’s charter is to define what Kubernetes native constructs are
+required to achieve these goals.
+
+Data Protection in Kubernetes includes backup and restore of applications
+to a working state. Part of the working group’s charter is to define what
+this requires.
+
+The Data Protection Working Group is organized with the goal of providing
+a cross SIG forum to discuss how to support data protection in Kubernetes,
+identify missing functionality, and work together to design features that
+are needed to achieve the goal.
+
+### In scope
+
+* High level discussions on what it means to support data protection in Kubernetes at different level and how to do it.
+* Design discussions on specific topics related to data protection and disaster recovery support.
+* Document results of discussions and investigations in a linkable medium.
+
+Potential design topics include, but are not limited to the following:
+* Extract snapshot data without cloning
+* Volume backups
+* Data populator
+* Retrieve diffs between two snapshots (block and file level)
```

Comment:
it seems a very low level functionality. Should it be part of kubernetes function?

Reply:
Without an kubernetes API to expose this, there is no easy way for backup vendor to retrieve this.  Lots of backup vendors requested this feature.  We should think about how to do it.